charts/springboot/templates/cronjobs.yaml

```diff
@@ -32,6 +32,35 @@ spec:
               - {{ . | replace "%FULLNAME%" (include "fullname" $) | quote }}
               {{- end }}
             {{- end }}
+            {{- if $definition.env }}
+            env:
+            {{- $secretName := include "secretname" $ -}}
+              {{- range $envname, $envdef := $definition.env }}
+              - name: {{ $envname | quote }}
+                {{- if $envdef.sensitive }}
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ $secretName | quote }}
+                    key: {{ $envdef.key | default $envname | quote }}
+                    {{- if $envdef.optional }}
+                    optional: true
+                    {{- end }}
+                {{- else if $envdef.configMapKeyRef }}
+                valueFrom:
+                  configMapKeyRef:
+{{ toYaml $envdef.configMapKeyRef | indent 20 }}
+                {{- else if $envdef.secretKeyRef }}
+                valueFrom:
+                  secretKeyRef:
+{{ toYaml $envdef.secretKeyRef | indent 20 }}
+                {{- else }}
+                value: {{ required (printf "Value for variable cronJobs.%s.env.%s.value is undefined" $name $envname) $envdef.value | quote }}
```

Comment:
Suggestion: pipe `$envdef.value` through `template` function in order to be flexible and support `{{ .Values.sth }}` expressions inside `values.yaml`. Same goes for deployment's env definition

Reply:
Make sense. Will do for both deployments & cronjobs in another PR

Reply:
@KarolisL what about being even more flexible and not breaking backwards compatibility https://github.com/inventi/helm/pull/8/ ?